pygmt/enums.py

```diff
@@ -0,0 +1,39 @@
+"""
+Enumerations for PyGMT.
+"""
+
+from enum import IntEnum
+
+
+class GridID(IntEnum):
```

Comment:
`GridID` doesn't sound very meaningful. How about `GridFormat`?

```suggestion
class GridFormat(IntEnum):
```